src/ploomber/util/default.py

```diff
@@ -257,6 +257,27 @@ def entry_point_relative(name=None):
     return location_pkg or location
 
 
+def try_to_find_env_yml(path_to_spec):
```

Comment:
Why don't we make it generic? pipeline.yml is also not supported in addition to env.yml